vendor/init.bat

```diff
@@ -160,7 +160,7 @@ if not defined HOME set "HOME=%USERPROFILE%"
 :: This is either a env variable set by the user or the result of
 :: cmder.exe setting this variable due to a commandline argument or a "cmder here"
 if defined CMDER_START (
-    cd /d "%CMDER_START%"
+    cd /d "%CMDER_START%\"
```

Comment:
It does not like being all on one line do this instead:

if defined CMDER_START (
    if "%CMDER_START:~-1%" neq "\" (
        SET CMDER_START="%CMDER_START%\"
    )

    cd /d "%CMDER_START%"
)

Make these changes and I will approve this change.